daemon/cmd/daemon.go

```diff
@@ -221,6 +222,9 @@ type Daemon struct {
 	// Tunnel-related configuration
 	tunnelConfig tunnel.Config
 	bwManager    bandwidth.Manager
+
+	// packet drop event emitter
+	dropEventEmitter *dropeventemitter.DropEventEmitter
```

Comment:
Any reason we need the dropEventEmitter in the `Daemon struct`? `Daemon` is already too big, and if we can let's avoid adding to it
```suggestion
```

Reply:
No, we don't. I thought we did but we don't.